bank/html-podstrony/bilety/oferta-biletow.html

```diff
@@ -11,54 +11,93 @@
-            <div class="ticket-wrapper">
-                <button id="MPK-N2" class="ticket"></button>
-                <span class="ticket-text"> MPK - normalny, 2 miesiące </span>
-            </div>
-            <div class="ticket-wrapper">
-                <button id="MPK-N3" class="ticket"></button>
-                <span class="ticket-text"> MPK - normalny, 3 miesiące </span>
-            </div>
+            <button class="ticket-wrapper">
+                <div id="MPK-U1" class="ticket">
+                    <img src="obrazki\MPK-U1.png" alt="Image Description">
+                </div>
+                <span class="ticket-text">MPK - ulgowy, 1 miesiąc<br></span>
+                <span class="ticket-text"> 79,50 zł </span>
+            </button>
+            <button class="ticket-wrapper">
+                <div id="MPK-U2" class="ticket">
+                    <img src="obrazki\MPK-U2.png" alt="Image Description">
+                </div>
+                <span class="ticket-text">MPK - ulgowy, 2 miesiące<br></span>
+                <span class="ticket-text"> 159,00 zł </span>
+            </button>
+            <button class="ticket-wrapper">
+                <div id="MPK-U3" class="ticket">
+                    <img src="obrazki\MPK-U3.png" alt="Image Description">
+                </div>
+                <span class="ticket-text">MPK - ulgowy, 3 miesiące<br></span>
+                <span class="ticket-text"> 238,50 zł </span>
+            </button>
+            <button class="ticket-wrapper">
+                <div id="MPK-N1" class="ticket">
+                    <img src="obrazki\MPK-N1.png" alt="Image Description">
+                </div>
+                <span class="ticket-text">MPK - normalny, 1 miesiąc<br></span>
+                <span class="ticket-text"> 159,00 zł </span>
+            </button>
+            <button class="ticket-wrapper">
+                <div id="MPK-N2" class="ticket">
+                    <img src="obrazki\MPK-N2.png" alt="Image Description">
+                </div>
+                <span class="ticket-text">MPK - normalny, 2 miesiące<br></span>
+                <span class="ticket-text">  318,00 zł </span>
+            </button>
+            <button class="ticket-wrapper">
+                <div id="MPK-N3" class="ticket">
+                    <img src="obrazki\MPK-N3.png" alt="Image Description">
+                </div>
+                <span class="ticket-text">MPK - normalny, 3 miesiące<br></span>
+                <span class="ticket-text"> 477,00 zł </span>
+            </button>
         </div>
     </div>
 
     <div id="choice-pr" class="choice">
         <div class="ticket-list">
+            <button class="ticket-wrapper">
+                <div id="P&R-KRK" class="ticket">
+                    <img src="obrazki\P+R_KRK.png" alt="Image Description">
+                </div>
+                <span class="ticket-text">Park & Ride Kraków<br></span>
+                <span class="ticket-text"> 10 zł </span>
```

Comment:
czy ta cena jest dla wszystkich pojazdów?
+
trzeba by było zaznaczyć, że to bilet na cały dzień, a nie na 2h (chyba, że jest inaczej, to mnie poprawcie)

Reply:
Z tego co patrzę na stronie to jest tylko rozróżnienie na bilet jednosobowy i grupowy, który kosztuje 25 zł. Można dodać poźniej też ten grupowy, a tu na razie dopisać, że to jednosobowy?
A co do czasu, to może trzeba by było dodać jakieś dodatkowe info przy stronie zakupu? I od razu to samo zrobić przy autostradach z osobówkami

Reply:
Pierwsze pytanie: myślę, że tak
Drugie pytanie: w którym miejscu byś to widziała? (mile widziana wizualizacja)